application/consumer-hazelcast-quarkus/src/test/java/webapp/tier/service/HazelcastSubscribeServiceTest.java

```diff
@@ -14,18 +23,32 @@
 	@Inject
 	HazelcastSubscribeService svc;
 
+	private static HazelcastInstance mockInstance;
+	String respbody = "message: Hello k8s-3tier-webapp with quarkus";
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6063083)